source/features/quick-comment-edit.tsx

```diff
@@ -7,6 +7,33 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import isArchivedRepo from '../helpers/is-archived-repo';
 
+function addQuickEditButton(commentForm: Element): void {
+	commentForm.classList.add('rgh-edit-comment');
+
+	const comment = commentForm.closest('.js-comment')!;
+	if (select.exists('.rgh-quick-comment-edit-button', comment)) { // #5572
+		return;
+	}
+
+	const button = (
+		<button
+			type="button"
+			role="menuitem"
+			className={`timeline-comment-action btn-link js-comment-edit-button ${pageDetect.isDiscussion() ? 'js-discussions-comment-edit-button' : ''} rgh-quick-comment-edit-button`}
+			aria-label="Edit comment"
+		>
+			<PencilIcon/>
+		</button>
+	);
+
+	const dropdown = comment.querySelector('.timeline-comment-actions > details:last-child')!;
```

Comment:
Need a child combinator in the selector, because the reaction picker on PR comments is also a `<details>`. (GitHub rolled some new reaction pickers, but on in PR conversations only :roll_eyes:)